src/renderer/styles/NumberInput.scss

```diff
@@ -41,9 +41,8 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
       outline: $spacing-xxs solid $white;
     }
 
-    &:disabled {
+    &[aria-disabled='true'] {
```

Comment:
Should we be shifting to using aria-disabled for all elements? Seems better to avoid using both disabled and aria-disabled across the code base

Reply:
Hmm looking at this more, correct me if im wrong, I guess we just use disabled for real html elements and aria-disabled for divs?

Reply:
yeah, it's cuz the `disabled` property actually DOES stuff to inputs while it means nothing on divs (since you can't interact with the actual HTML element itself). If we wanted to be more consistent, we could choose to either be very explicit with this divide that you described, or just slap em both onto everything